docs/testbed/README.testbed.IxANVL.VsSetup.md

```diff
@@ -0,0 +1,158 @@
+# Testbed Setup with Keysight IxANVL container and SONIC Virtual DUT
+ 
+ This document describes the steps to setup a testbed with Keysight IxANVL container and SONIC Virtual DUT to run BGP conformance test cases. 
+ 
+ ### Testbed server 
+ The schematic diagram below provides an overview of the setup. 
+ ![](img\keysight-ixanvl-testbed-topology.png) 
```

Comment:
Schematic diagram url is not loading, replace `-` with `_`

Reply:
Changed the name as replace - with _.